isamples_metadata/GEOMETransformer.py

```diff
@@ -143,18 +144,18 @@ def sample_description(self) -> str:
 
         return Transformer.DESCRIPTION_SEPARATOR.join(description_pieces)
 
-    def has_context_categories(self) -> list:
+    def has_context_categories(self) -> list[VocabularyTerm]:
         # TODO: resolve https://github.com/isamplesorg/isamples_inabox/issues/312
         # This should probably return the biological kingdom once that is hooked into the vocabulary
-        return [vocabulary_mapper.SAMPLED_FEATURE.term_for_key("sf:marinewaterbody").metadata_dict()]
+        return [vocabulary_mapper.sampled_feature_type().term_for_key("sf:marinewaterbody")]
```

Comment:
One annoying fallout of moving things to the database (these used to reference the text file dumps @smrgeoinfo provided me), is that we can't reference static variables.  Now, a pedantic programmer might point out that static variables are a poor design choice, and they wouldn't be wrong, but it *was* simpler before.  :)

Reply:
So, the replacement is a function that fetches them from the database (or if they've been fetched already, out of a cache).